egd_hr_timesheet_custom/README.rst

```diff
@@ -0,0 +1,54 @@
+=======================
+Egd Hr Timesheet Custom
```

Comment:
O nome do aplicativo, acho que faz sentido indicar que é a inclusão do Tracking do valor de Custo Hora

Sugestão:
Hr Timesheet adding Tracking to timesheet_cost Field

Esse aplicativo poderá ser usado em outros clientes...